src/parser/jobs/nin/Ninjutsu.js

```diff
@@ -0,0 +1,64 @@
+import React, {Fragment} from 'react'
+//import {Icon, Message} from 'semantic-ui-react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+//import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class Ninjutsu extends Module {
+	static handle = 'ninjutsu'
+	static dependencies = [
+		'suggestions',
+	]
+
+	_hyotonCount = 0
+	_rabbitCount = 0
+
+	constructor(...args) {
+		super(...args)
+		this.addHook('cast', {by: 'player'}, this._onCast)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onCast(event) {
+		const abilityId = event.ability.guid
+
+		if (abilityId === ACTIONS.HYOTON.id) {
+			this._hyotonCount++
+		}
+
+		if (abilityId === ACTIONS.RABBIT_MEDIUM.id) {
```

Comment:
Not an issue, but just saying: _totes_ should track critbunnies too :blobknife:

Hah.

Reply:
:thinkpaint: